### PR TITLE
Docs: updating installation sections.

### PR DESCRIPTION
**Pull request by godai78**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Docs: updating installation sections. Closes cksource/ckeditor5-commercial#6076

---

### Additional information

_Paired with_ cksource/ckeditor5-commercial#6080

### CODE REVIEW

File: packages/ckeditor5-typing/docs/features/text-transformation.md [hunk 1]
```diff
@@ -78,72 +78,68 @@ By using the options defined below you can extend, limit, or override this list:
...
+				// Plus some custom transformation.
+				{ from: 'CKE', to: 'CKEditor' }
+			],
 		}
-	} )
-	.then( /* ... */ )
-	.catch( /* ... */ );
+	}
+} )
+	.then( /* ... */ );
```
Comment (gorzelinski) [inline packages/ckeditor5-typing/docs/features/text-transformation.md:95]:
```suggestion
.then( /* ... */ );
```

File: packages/ckeditor5-typing/docs/features/text-transformation.md [hunk 2]
```diff
@@ -78,72 +78,68 @@ By using the options defined below you can extend, limit, or override this list:
...
+					to: matches => [ null, matches[ 1 ].toUpperCase() ]
+				}
+			],
 		}
-	} )
-	.then( /* ... */ )
-	.catch( /* ... */ );
+	}
+} )
+	.then( /* ... */ );
```
Comment (gorzelinski) [inline packages/ckeditor5-typing/docs/features/text-transformation.md:142]:
```suggestion
.then( /* ... */ );
```

File: packages/ckeditor5-typing/docs/features/text-transformation.md [hunk 3]
```diff
@@ -154,23 +150,15 @@ You can test these custom rules in the demo. Try typing `:)` or `:+1:` and see h
...
-		plugins: [ TextTransformation, ... ],
-	} )
-	.then( /* ... */ )
-	.catch( /* ... */ );
+import { ClassicEditor, TextTransformation } from 'ckeditor5';
+
+ClassicEditor.create( document.querySelector( '#editor' ), {
+	plugins: [ TextTransformation, ... ],
+} )
+	.then( /* ... */ );
```
Comment (gorzelinski) [inline packages/ckeditor5-typing/docs/features/text-transformation.md:161]:
```suggestion
.then( /* ... */ );
```

File: packages/ckeditor5-upload/docs/features/base64-upload-adapter.md [hunk 4]
```diff
@@ -29,24 +29,16 @@ Use the editor below to see the adapter in action. Open the web browser console
...
-	} )
-	.then( /* ... */ )
-	.catch( /* ... */ );
+import { ClassicEditor, Base64UploadAdapter } from 'ckeditor5';
+
+ClassicEditor.create( document.querySelector( '#editor' ), {
+	plugins: [ Base64UploadAdapter, /* ... */ ],
+	toolbar: [ /* ... */ ]
+} )
+	.then( /* ... */ );
```
Comment (gorzelinski) [inline packages/ckeditor5-upload/docs/features/base64-upload-adapter.md:41]:
```suggestion
.then( /* ... */ );
```

File: packages/ckeditor5-upload/docs/features/simple-upload-adapter.md [hunk 5]
```diff
@@ -11,27 +11,19 @@ The simple upload adapter lets you upload images to your server using the [`XMLH
...
+import { ClassicEditor, SimpleUploadAdapter } from 'ckeditor5';
+
+ClassicEditor.create( document.querySelector( '#editor' ), {
+	plugins: [ SimpleUploadAdapter, /* ... */ ],
+	toolbar: [ /* ... */ ],
+	simpleUpload: {
+		// Feature configuration.
+	}
+} )
+	.then( /* ... */ );
```
Comment (gorzelinski) [inline packages/ckeditor5-upload/docs/features/simple-upload-adapter.md:26]:
```suggestion
.then( /* ... */ );
```

File: packages/ckeditor5-watchdog/docs/features/watchdog.md [hunk 6]
```diff
@@ -41,11 +41,11 @@ Then, change your `ClassicEditor.create()` call to `watchdog.create()` as follow
...
 import { ClassicEditor } from '@ckeditor/ckeditor5-editor-classic';
-import { EditorWatchdog } from '@ckeditor/ckeditor5-watchdog';
+import { ClassicEditor, EditorWatchdog } from '@ckeditor/ckeditor5-watchdog';
 
-import { Bold, Italic } from '@ckeditor/ckeditor5-basic-styles';
-import { Essentials } from '@ckeditor/ckeditor5-essentials';
-import { Paragraph } from '@ckeditor/ckeditor5-paragraph';
+import { ClassicEditor, Bold, Italic } from '@ckeditor/ckeditor5-basic-styles';
+import { ClassicEditor, Essentials } from '@ckeditor/ckeditor5-essentials';
+import { ClassicEditor, Paragraph } from '@ckeditor/ckeditor5-paragraph';
```
Comment (gorzelinski) [inline packages/ckeditor5-watchdog/docs/features/watchdog.md:48]:
```suggestion
import { ClassicEditor, Bold, EditorWatchdog, Essentials, Italic, Paragraph } from 'ckeditor5';
```

I guess something went wrong here after some mechanical changes. I also guess that EditorWatchdog will be available in the open-source package.

File: packages/ckeditor5-word-count/docs/features/word-count.md [hunk 7]
```diff
@@ -32,10 +32,9 @@ The example above was created by using the following HTML page structure:
...
-ClassicEditor
-	.create( document.querySelector( '#editor' ), {
-		// Configuration details.
-	} )
+ClassicEditor.create( document.querySelector( '#editor' ), {
+	// Configuration details.
+} )
 	.then( editor => {
 		const wordCountPlugin = editor.plugins.get( 'WordCount' );
 		const wordCountWrapper = document.getElementById( 'word-count' );
```
Comment (gorzelinski) [inline packages/ckeditor5-word-count/docs/features/word-count.md:40]:
```suggestion
.then( editor => {
	const wordCountPlugin = editor.plugins.get( 'WordCount' );
	const wordCountWrapper = document.getElementById( 'word-count' );
```

File: packages/ckeditor5-word-count/docs/features/word-count.md [hunk 8]
```diff
@@ -232,23 +229,15 @@ Here is the HTML structure used to create the customized word and character coun
...
-ClassicEditor
-	.create( document.querySelector( '#editor' ), {
-		plugins: [ WordCount, /* ... */ ],
-	} )
-	.then( /* ... */ )
-	.catch( /* ... */ );
+ClassicEditor.create( document.querySelector( '#editor' ), {
+	plugins: [ WordCount, /* ... */ ],
+} )
+	.then( /* ... */ );
```
Comment (gorzelinski) [inline packages/ckeditor5-word-count/docs/features/word-count.md:240]:
```suggestion
.then( /* ... */ );
```

**Review by gorzelinski — CHANGES_REQUESTED** [review]
Most of the changes are probably done semi-automatically (I guess), and that's good. But some of the formatting is off because of that. I would change the formatting before merging.

File: packages/ckeditor5-alignment/docs/features/text-alignment.md [hunk 9]
```diff
@@ -31,17 +31,15 @@ It is possible to configure which alignment options are available in the editor
...
-	.catch( /* ... */ );
+ClassicEditor.create( document.querySelector( '#editor' ), {
+	alignment: {
+		options: [ 'left', 'right' ]
+	},
+	toolbar: [
+		'heading', '|', 'bulletedList', 'numberedList', 'alignment', 'undo', 'redo'
+	]
+} )
+	.then( /* ... */ );
```
[the remark this answers is not in the document]
Comment (godai78) [reply packages/ckeditor5-alignment/docs/features/text-alignment.md:42]: It was done along the lines of accepted version from Notion...  
  
[https://www.notion.so/Implement-changes-to-basic-Features-guides-071c814c3dc447b49b3255f64da15875](https://www.notion.so/Implement-changes-to-basic-Features-guides-071c814c3dc447b49b3255f64da15875)  
  
Too bad the accepted version was not correct :D

File: packages/ckeditor5-watchdog/docs/features/watchdog.md [hunk 10]
```diff
@@ -41,11 +41,9 @@ Then, change your `ClassicEditor.create()` call to `watchdog.create()` as follow
 
 ```js
 import { ClassicEditor } from '@ckeditor/ckeditor5-editor-classic';
-import { EditorWatchdog } from '@ckeditor/ckeditor5-watchdog';
+import { ClassicEditor, EditorWatchdog } from '@ckeditor/ckeditor5-watchdog';
```
Comment (gorzelinski) [inline packages/ckeditor5-watchdog/docs/features/watchdog.md:44]:
```suggestion

```

File: packages/ckeditor5-watchdog/docs/features/watchdog.md [hunk 11]
```diff
@@ -41,11 +41,9 @@ Then, change your `ClassicEditor.create()` call to `watchdog.create()` as follow
 
 ```js
 import { ClassicEditor } from '@ckeditor/ckeditor5-editor-classic';
```
Comment (gorzelinski) [inline packages/ckeditor5-watchdog/docs/features/watchdog.md:43]:
```suggestion

```

File: packages/ckeditor5-watchdog/docs/features/watchdog.md (hunk 10 above)
Comment (gorzelinski) [reply packages/ckeditor5-watchdog/docs/features/watchdog.md:44]: Those imports are no longer needed - everything is imported from the main open-source package.

File: packages/ckeditor5-word-count/docs/features/word-count.md [hunk 12]
```diff
@@ -32,13 +32,12 @@ The example above was created by using the following HTML page structure:
...
-		const wordCountWrapper = document.getElementById( 'word-count' );
+ClassicEditor.create( document.querySelector( '#editor' ), {
+	// Configuration details.
+} )
+.then( editor => {
+	const wordCountPlugin = editor.plugins.get( 'WordCount' );
+	const wordCountWrapper = document.getElementById( 'word-count' );
 
 		wordCountWrapper.appendChild( wordCountPlugin.wordCountContainer );
 	} )
```
Comment (gorzelinski) [inline packages/ckeditor5-word-count/docs/features/word-count.md:43]:
```suggestion
ClassicEditor.create( document.querySelector( '#editor' ), {
	// Configuration details.
} )
.then( editor => {
	const wordCountPlugin = editor.plugins.get( 'WordCount' );
	const wordCountWrapper = document.getElementById( 'word-count' );

	wordCountWrapper.appendChild( wordCountPlugin.wordCountContainer );
} )
```

File: packages/ckeditor5-word-count/docs/features/word-count.md
Comment (gorzelinski) [inline packages/ckeditor5-word-count/docs/features/word-count.md:1]: There are still `catch()` blocks in this guide.

**Review by gorzelinski — COMMENTED** [review]
I found some minor problems - mainly with some imports.

File: packages/ckeditor5-ckfinder/docs/features/ckfinder.md
[the remark this answers is not in the document]
Comment (godai78) [reply packages/ckeditor5-ckfinder/docs/features/ckfinder.md:1]: In commit.

File: packages/ckeditor5-word-count/docs/features/word-count.md
Comment (godai78) [reply packages/ckeditor5-word-count/docs/features/word-count.md:1]: In commit.

File: packages/ckeditor5-language/docs/features/language.md [hunk 13]
```diff
@@ -32,43 +32,33 @@ To modify the list of available languages displayed in the language dropdown use
...
-    .catch( /* ... */ );
+ClassicEditor.create( document.querySelector( '#editor' ), {
+	// More of editor's configuration.
+	// ...
+	language: {
+		textPartLanguage: [
+			{ title: 'Arabic', languageCode: 'ar' },
+		{ title: 'French', languageCode: 'fr' },
+			{ title: 'Hebrew', languageCode: 'he' },
+			{ title: 'Spanish', languageCode: 'es' }
```
Comment (filipsobol) [inline packages/ckeditor5-language/docs/features/language.md:43]:
```suggestion
			{ title: 'Arabic', languageCode: 'ar' },
			{ title: 'French', languageCode: 'fr' },
			{ title: 'Hebrew', languageCode: 'he' },
			{ title: 'Spanish', languageCode: 'es' }
```

File: packages/ckeditor5-list/docs/features/lists-editing.md [hunk 14]
```diff
@@ -23,40 +23,34 @@ To edit a block inside a list item, press <kbd>Enter</kbd> to create a new line
...
-			/* Other plugins. */
-		],
-	} )
-	.then( /* ... */ )
-	.catch( /* ... */ );
+import { ClassicEditor, List, AdjacentListsSupport } from 'ckeditor5';
+
+ClassicEditor.create( document.querySelector( '#editor' ), {
+	plugins: [ List, AdjacentListsSupport, /* ... */
+	],
```
Comment (filipsobol) [inline packages/ckeditor5-list/docs/features/lists-editing.md:51]:
```suggestion
	plugins: [ List, AdjacentListsSupport, /* ... */ ],
```

File: packages/ckeditor5-minimap/docs/features/minimap.md [hunk 15]
```diff
@@ -110,48 +110,37 @@ Employ the following CSS:
...
+	minimap: {
+		container: document.querySelector( '.minimap-container' ),
+	}
+} )
+.then( editor => {
+	const toolbarContainer = document.querySelector( '#toolbar-container' );
+
+	toolbarContainer.appendChild( editor.ui.view.toolbar.element );
+} )
+.catch( /* ... */ );
```
Comment (filipsobol) [inline packages/ckeditor5-minimap/docs/features/minimap.md:126]:
```suggestion
} );
```

**Pull request merged**